unix/gles2.cpp

```diff
@@ -318,19 +318,34 @@ void gles2_create(int display_width, int display_height, int bitmap_width, int b
 	dis_width = display_width;
 	dis_height = display_height;
 
-	// Screen aspect ratio adjustment
-	if(Settings.MaintainAspectRatio)
+    if(Settings.DisplayInteger)
+    {
+        // screen integer - x1
+	    sx = (float)bitmap_width/(float)display_width;
+	    sy = (float)bitmap_height/(float)display_height;
+
+	    SetOrtho(proj, -0.5f, +0.5f, +0.5f, -0.5f, -1.0f, 1.0f, sx, sy);
+
+	    printf("integer enabled - %ux%u : %ux%u (x:%f y:%f)\n", 
+	        display_width, display_height, bitmap_width, bitmap_height, sx, sy);
+    }
+    else
 	{
```

Comment:
I think the original indentation is somewhat wonky but can you align this as the original and we can reformat later if needed. Thanks.

Reply:
You preffer *printf* in just one line?

I usually try to adapt my code to 80cols, but no problem.

I fixed some missing tabs in my code, thanks @joolswills